app/controlplane/api/controlplane/v1/context.proto

```diff
@@ -32,7 +32,7 @@ message ContextServiceCurrentResponse {
   
   message Result {
     User current_user = 1;
-    Org current_org = 2;
+    OrgItem current_org = 2;
```

Comment:
this change it just to be consistent with the name of other resources in the proto defs